### DISCUSSION TITLE
Icon Proposal - Farcaster

### DISCUSSION THREAD

**Discussion by alexcbruno** [phosphor-icons/homepage#435] [Ideas]
![Farcaster-Icon-Proposal-Cover-0 1](https://github.com/phosphor-icons/homepage/assets/7109208/1abcbfd4-10b3-4a78-9569-b074cc5e9c2a)

## About
When building my website, I hoped Phosphor might have an icon I could use to link to my Farcaster account. I couldn't find something that would suffice, so I'm working on one and would like to propose that it be included in the official library.

Relevant links:
- [Figma community file](https://www.figma.com/community/file/1343744155174419800/farcaster-icon-proposal)
- [Farcaster homepage](http://farcaster.xyz)

## Icon options
Three versions of the icon have been drafted. They can be seen below. Feedback from the Phosphor Icons team regarding which option best suits the library would be great!
![Farcaster-Icon-Options-0 1a](https://github.com/phosphor-icons/homepage/assets/7109208/d7fed5d9-f1a4-43fc-95ab-edc2f76afcf3)

## Icon design
Icons were designed using the Farcaster logo and other Phosphor icons as reference. I based the icon key lines grid off a [Medium post from the illustrious Helena Zhang](https://minoraxis.medium.com/icon-grids-keylines-demystified-5a228fe08cfd). Icons were designed in Figma and published as a [community file](https://www.figma.com/community/file/1343744155174419800/farcaster-icon-proposal).

## Iteration
I’m happy to accept critique and changes from the Phosphor Icons team and implement edit based on their feedback. I’m equally happy to hand over files and creative control to Phosphor Icons team!

## Addressing points regarding new icon requests
> 1. Represents a social network or platform, or an open standard

The icon, while it could be used generically to reference an architectural arch, would also represent [Farcaster](http://farcaster.xyz), a “sufficiently decentralized social network built on Ethereum.” Farcaster is a social-networking protocol upon which other platforms can build. The most widely used platform is [Warpcast](http://warpcast.com) which is built and maintained by the Farcaster team.

> 2. Has large reach and recognition, and is not specific to one country

Farcaster currently serves 300,000+ users across the globe, many of whom are developers and designers for crypto, web3, and tech products. It’s open for anyone to create an account, and is currently growing its user base by ~12% per week. See [other stats on the network](http://farcaster.network).

> 3. Is useful outside of commercial contexts (a link on a personal website or portfolio, for example)

Although the genesis of this proposal was my portfolio page, the icon would certainly have usage outside of portfolio links. Farcaster is a social networking protocol, similar to HTTP for the web and SMTP for email. Apart from representing the social network, the icon could be used to reference architectural arches.

## To address legal concerns
I reached out to Farcaster co-founder Dan Romero for a temp check on this proposal, to which he replied `Love it go for it!`
Farcaster’s builder culture is evident when browsing the platform, with many products and users modifying the logo.

## End
I hope this proposal eases the inclusion of a new icon into an already fantastic library. Eager to hear the team's feedback and if there are next steps.

**Comment by rektdeckard** [comment]
@alexcbruno thanks for the incredibly thoughtful and thorough proposal! I honestly have no notes on the icon design itself. If we had a spec, this would be 100% compliant 😄.

Unfortunately, we didn't feel that it was right to add to Phosphor (right now). There's a few reason for this:

1. I'm not sure it really has the reach or recognition we look for, yet. The numbers I found [[1]](https://dune.com/pixelhack/farcaster)[[2]](https://farcaster.network/) looked more like 300K total signups, not active users. And the vast majority of that in January and February. I don't know if it's something that's here to stay, or disappears next year.
2. Farcaster is a protocol, not an app; it is that much less likely the average Phosphor user will need this icon, ever. What we choose to add to Phosphor is as important as what we choose to leave out — we have to think about how many bytes we ship to clients that don't have a build step, and in general prefer small-and-tight to boil-the-ocean.
3. As far as I could tell, a fair amount of the content seems to be centered around meme/scam coins, shilling, etc. I'm not here to judge, and am definitely not very informed, but I do think that an icon is an implicit endorsement.
4. We are trying to move away from logos and brand icons as much as possible, for reasons I'm sure you picked up on by reading #392. 

For now, it's simply too soon. In the mean time, we do have first-class support for [custom icons in our react](https://github.com/phosphor-icons/react?tab=readme-ov-file#custom-icons) and [elm](https://package.elm-lang.org/packages/phosphor-icons/phosphor-elm/latest/Phosphor#customIcon) libraries. Happy to wait and be proven wrong on points 1-3!

Thanks again for putting in the time.
